LiveVox.NET/Models/Call/Enumerations/ErrorCode.cs

```diff
@@ -0,0 +1,132 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+
+namespace LiveVox.NET.Models.Call.Enumerations
+{
+    public enum ErrorCode
+    { 
+        /// <summary>
+        /// Error code 440: Busy
+        /// </summary>
+        [JsonPropertyName("440")]
```

Comment:
@srs-adamr here we have two properties with the same value Busy, 
the same for OutsideOfAllowableDialingWindow too.